setup.py

```diff
@@ -65,7 +65,7 @@ def get_cuda_version():
     "kornia",
     "lightning",
     "matplotlib",
-    "moviepy",
+    "moviepy<2.0.0", # update is to be tested
```

Comment:
2.0.0 was released recently, pretty sure our moviepy usage will break if you upgrade. clip.fl method is called something else in 2.0.0